docs/pages/guides/extending-a-world/index.mdx

```diff
@@ -780,6 +780,16 @@ We will now fix this.
        client: { public: publicClient, wallet: burnerWalletClient },
      });
 
+     /* Update counterConfig.tables so it'll work as components */
+     const resolvedTables = storeToV1(counterConfig).tables;
```

Comment:
This isn't meant to be used externally and will be removed very soon.

Reply:
We'll have another mechanism to add components to the synchronization?

Reply:
yes, you can pass config output's tables directly into `syncToRecs`

Reply:
I assume you mean:

```
import counterConfig from "./counter.config";
.
.
.
  const { components, latestBlock$, storedBlockLogs$, waitForTransaction } = await syncToRecs({
    world,
    config: mudConfig,
    address: networkConfig.worldAddress as Hex,
    publicClient,
    startBlock: BigInt(networkConfig.initialBlockNumber),
    tables: counterConfig.tables,
  });
```

I tried it, but it dies in the `syncToRecs` call with this error:

```
TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at U4 (chunk-ACLFEHT3.js?v=0ed45cff:7222:48)
    at chunk-ACLFEHT3.js?v=0ed45cff:7225:24
    at chunk-A6CZEM5O.js?v=0ed45cff:48:69
    at Array.map (<anonymous>)
    at P (chunk-A6CZEM5O.js?v=0ed45cff:48:48)
    at I4 (chunk-ACLFEHT3.js?v=0ed45cff:7225:10)
    at Q2 (chunk-ACLFEHT3.js?v=0ed45cff:7231:17)
    at Qe (chunk-ACLFEHT3.js?v=0ed45cff:7267:86)
    at setupNetwork (setupNetwork.ts:98:25)
```

Or do you mean the output not from `defineWorld` but something else?

Reply:
ahh this is changed in main but not yet released, just waiting for multi namespaces (almost here, prob this week)